src/utils/Processing.ts

```diff
@@ -284,14 +285,16 @@ export function init(): void {
     path.join(outputDir, '.gitignore')
   );
   // Add the _updatePublisher, _genonce, and _gencontinuous scripts
-  const scriptsDir = path.join(__dirname, '..', 'ig', 'files');
-  const scriptsDirContents = fs.readdirSync(scriptsDir);
-  scriptsDirContents
-    .filter(file => file.startsWith('_'))
-    .forEach(file => {
-      fs.copyFileSync(path.join(scriptsDir, file), path.join(outputDir, file));
-    });
-
+  console.log('Downloading publisher scripts from https://github.com/FHIR/sample-ig');
+  for (const script of [
+    '_genonce.bat',
+    '_genonce.sh',
+    '_updatePublisher.bat',
+    '_updatePublisher.sh'
+  ]) {
+    const res = await axios.get(`http://raw.githubusercontent.com/FHIR/sample-ig/master/${script}`);
```

Comment:
It looks to me like if the download fails, an error will be thrown and the function will bail here.  I think we want to wrap this in try/catch so if a download fails, we can inform the user (and maybe even direct them to the URL for manual download), and keep on processing the init.

It would be sad if someone is in a locked down environment, or has weird proxies or something, and can't run SUSHI init at all.